[Bug] Expand GitHub error message (#13671)
demisto/content: Packs/GitHub/ReleaseNotes/1_3_1.md

#### Integrations
##### GitHub

> - Added to commands error messages the GitHub error message when applicable.

Reviewer: ```suggestion
- Improved the error messages from GitHub upon a failure when applicable.
```
